templates/registration/done.html

```diff
@@ -46,42 +46,33 @@ <h1 class="title-bold-red">#ENDTHEWAITLIST</h1>
         <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-24">
             <p class="done-text-bold margin-none">
                 {% blocktrans trimmed %}
-                    Each registered donor can save up to 8 lives.
+                    Spread the word…
                 {% endblocktrans %}
             </p>
             <p class="done-text-regular margin-none">
                 {% blocktrans trimmed %}
-                    Telling your friends to register at ORGANIZE is just another way you can help us #EndTheWaitlist
+                    It's important that your friends and family know your donor wishes, so spread the word by tweeting "I am an
+                    #OrganDonor" today.
                 {% endblocktrans %}
             </p>
         </div>
     </div>
 
-    <div class="row">
-        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">&nbsp;</div>
-        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">&nbsp;</div>
-    </div>
-
-    {% if FACEBOOK_APP_ID %}
-        <div class="row" style="margin-bottom: 60px;">
-            <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
-
-                <img src="{% static 'images/fb-icon.svg' %}" class="done-social-icon done-fb-icon"/>
-                <a href="" onclick="fb_share();" class="btn btn-default btn-organize done-btn-social done-btn-share">
-                    {% blocktrans trimmed %}
-                        Share
-                    {% endblocktrans %}
-                </a>
-
-            </div>
+    <div class="row" style="margin: 40px 0;">
+        <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+            <video id="organizeVideo" width="90%" controls="controls">
```

Comment:
I think this should be 100%. Is there a reason why you did this? If its necessary, please center the video.